packages/desktop-client/src/components/transactions/TransactionsTable.js

```diff
@@ -835,7 +835,8 @@ const Transaction = memo(function Transaction(props) {
         isPreview && { color: colors.n5, fontStyle: 'italic' },
         _unmatched && { opacity: 0.5 },
       ]}
-      onMouseEnter={() => onHover && onHover(transaction.id)}
+      onMouseEnter={() => !isNew && setHovered(true)}
```

Comment:
Maybe we can start moving to pointer events?
```suggestion
      onPointerEnter={() => !isNew && setHovered(true)}
```